otter/http.py

```diff
@@ -145,11 +155,105 @@ def service_request(
         data=data,
         log=log,
         reauth_codes=reauth_codes,
-        success_codes=success_codes,
+        success_pred=success_pred,
         json_response=json_response))
 
 
 @attributes(["service_type", "method", "url", "headers", "data",
-             "log", "reauth_codes", "success_codes", "json_response"])
+             "log", "reauth_codes", "success_pred", "json_response"])
 class ServiceRequest(object):
-    """A request to a Rackspace/OpenStack service."""
+    """
+    A request to a Rackspace/OpenStack service.
+
+    Note that this intent does _not_ contain a tenant ID. To specify the tenant
+    ID for any tree of effects that might contain a ServiceRequest, wrap the
+    effect in a :obj:`TenantScope`.
+    """
+
+
+@attributes(['effect', 'tenant_id'], apply_with_init=False)
+class TenantScope(object):
+    """
+    An intent that specifies a tenant for any effect which might make
+    :obj:`ServiceRequest`s.
+
+    In other words, something like this::
+
+        perform(dispatcher, TenantScope(Effect(ServiceRequest(...))))
+
+    will make the ServiceRequest bound to the specified tenant.
```

Comment:
I'm not totally sure what you mean, but I can extend the docs here in a few reasonable ways. (no intents introduce behavior; they all require performers in dispatchers. in this case, though, I agree it's a good idea to explicitly call out the performer since you'd only ever want to use one performer)